kouta-backend/src/main/scala/fi/oph/kouta/util/NameHelper.scala

```diff
@@ -110,4 +111,24 @@ object NameHelper {
       hakukohdeNimi
     }
   }
+
+  def nimiOrEmptyString(nimi: Option[String]): String = {
+    nimi match {
+      case Some(str) => str
+      case None => ""
+    }
+  }
```

Comment:
Eikös tän sijaan voi käyttää vaan nimi.getOrElse("")

Reply:
Poistin koko apufunktion. Ei meinaa vielä muistaa näitä Scalan kikkoja.